cookiecutter.json

```diff
@@ -9,6 +9,7 @@
     "plugin_name": "{{ cookiecutter.project_name|replace(' ', '') }}",
     "repo_name": "{{ cookiecutter.project_name|lower|replace(' ', '') }}",
     "plugin_uri": "https://{{ cookiecutter.domain }}/plugins/{{ cookiecutter.repo_name }}",
+    "plugin_clap_id": "com.github.{{ cookiecutter.github_username }}.{{ cookiecutter.repo_name }}",
```

Comment:
This part could be improved.

The CLAP ID is meant to be a reverse URI. Technically, we should probably convert `cookiecutter.plugin_uri` into a reverse URI.

For the time being, to keep things simple, I went with this kind of default value instead: `com.github.pdesaulniers.simple-amp` (which implies that the plugin is associated with the url `github.com/pdesaulniers/simple-amp`)

Reply:
Yeah, let's go with that for now.